docs/customizations/custom-server.md

```
@@ -0,0 +1,77 @@
+    },
+})
+</script>
+
+<OASpec />
+```
+
+</template>
+
+</ScopeConfigurationTabs>
+
+<ExampleBlock>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: ExampleBlock

(MD033, no-inline-html)